spec/src/main/asciidoc/images/xmlb-4.svg

```diff
@@ -0,0 +1,175 @@
+		<g id="shape53-33" v:mID="53" v:groupContext="shape" transform="translate(139.901,-72.4365) scale(-1,1)">
+			<title>Foglio.53</title>
+			<path d="M5.67 236.72 L0 236.72 L2.83 227.08 L5.67 236.72 Z" class="st9"/>
+		</g>
+		<g id="shape29-35" v:mID="29" v:groupContext="shape" transform="translate(333.222,311.052) rotate(157.299)">
+			<title>Foglio.29</title>
+			<path d="M0 236.72 L61.04 236.72" class="st8"/>
+		</g>
+		<g id="shape30-38" v:mID="30" v:groupContext="shape" transform="translate(454.079,12.5135) rotate(70)">
+			<title>Foglio.30</title>
+			<path d="M5.67 236.72 L0 236.72 L2.83 227.08 L5.67 236.72 Z" class="st9"/>
+		</g>
+		<g id="shape54-40" v:mID="54" v:groupContext="shape" transform="translate(150.595,373.616) rotate(-157.299)">
+			<title>Foglio.54</title>
+			<path d="M0 236.72 L61.04 236.72" class="st8"/>
+		</g>
+		<g id="shape55-43" v:mID="55" v:groupContext="shape" transform="translate(456.07,230.21) rotate(110)">
+			<title>Foglio.55</title>
+			<path d="M5.67 236.72 L0 236.72 L2.83 227.08 L5.67 236.72 Z" class="st9"/>
+		</g>
+		<g id="shape25-45" v:mID="25" v:groupContext="shape" transform="translate(245.999,-196.311)">
+			<title>Foglio.25</title>
+			<rect x="0" y="202.295" width="197.209" height="34.4207" class="st3"/>
+		</g>
+		<g id="shape28-47" v:mID="28" v:groupContext="shape" transform="translate(275.964,-204.005)">
+			<title>Foglio.28</title>
+			<desc>Application Code</desc>
+			<v:textBlock v:margins="rect(4,4,4,4)" v:tabSpace="42.5197"/>
+			<v:textRect cx="46.7717" cy="227.807" width="93.55" height="17.8178"/>
+			<rect x="0" y="218.898" width="93.5433" height="17.8178" class="st4"/>
+			<text x="4" y="230.81" class="st5" v:langID="1040"><v:paragraph/><v:tabList/>Application Code</text>		</g>
+		<g id="shape64-50" v:mID="64" v:groupContext="shape" transform="translate(245.796,-122.61)">
+			<title>Foglio.64</title>
+			<rect x="0" y="163.015" width="101.035" height="73.7008" class="st3"/>
+		</g>
+		<g id="shape65-52" v:mID="65" v:groupContext="shape" transform="translate(246.306,-149.539)">
+			<title>Foglio.65</title>
+			<desc>Schema Derived Interfaces, Factory Methods</desc>
+			<v:textBlock v:margins="rect(4,4,4,4)" v:tabSpace="42.5197"/>
+			<v:textRect cx="39.579" cy="219.708" width="79.16" height="34.0157"/>
+			<rect x="0" y="202.7" width="79.158" height="34.0157" class="st4"/>
+			<text x="4" y="212.51" class="st7" v:langID="1040"><v:paragraph/><v:tabList/>Schema Derived <tspan x="4" dy="1.2em"
+						class="st6">Interfaces</tspan>,<v:newlineChar/><tspan x="4" dy="1.2em" class="st6">Factory Methods</tspan></text>		</g>
+		<g id="shape66-57" v:mID="66" v:groupContext="shape" transform="translate(346.83,-122.918)">
+			<title>Foglio.66</title>
+			<rect x="0" y="163.632" width="96.378" height="73.084" class="st3"/>
+		</g>
+		<g id="shape67-59" v:mID="67" v:groupContext="shape" transform="translate(349.163,-150.956)">
+			<title>Foglio.67</title>
+			<desc>Package Javax.xml.bind</desc>
```

Comment:
package name should be all lowercase

Reply:
Yes, you are right! I correct it right now.

Reply:
Now package name is in lowercase.
Sorry for typo and thank you!
Dmitri